.pre-commit-config.yaml

```diff
@@ -27,13 +27,13 @@ repos:
         exclude: (.bumpversion.cfg|yarn.js)
 
   - repo: https://github.com/python-jsonschema/check-jsonschema
-    rev: 0.29.4
+    rev: 0.30.0
     hooks:
       - id: check-github-workflows
 
   # Check ruff version is aligned with the one in pyproject.toml
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.7.2
+    rev: v0.8.1
```

Comment:
CI passes but we should look into updating `ruff` here too:

https://github.com/jupyterlab/jupyterlab/blob/69b792511c810666ea0c79836e92be850816f00e/pyproject.toml#L122